docs/source/project-management/work-management.rst

```diff
@@ -0,0 +1,83 @@
+.. _work-management:
+
+Work Management
+===============
+
+
+Work Management Overview
+------------------------
+
+The SDC uses Github issues to track work. These issues are organized into two types of tasks: high-level tasks and implementation tasks.
+
+- High-level tasks
+    - used to track the progress of each instrument/data level and infrastructure features (ie. Reprocessing high-level task)
+    - tasks should be fairly static throughout development and will only change if an instrument changes their data products or levels
+    - used to track the progress of implementation tasks
+    - managed in the high-level board
+    - labeled with the "Repo: Parent Issue" GitHub label
+- Implementation tasks
+    - define work that needs to be implemented
+    - each task should be small enough to be handled in a single PR
+    - organized as sub-tasks under high-level tasks
+    - should be added to high-level tasks as soon as they are identified
+    - should have work estimates tied to each task
+    - managed in the release boards
```

Comment:
Do we need each task to be part of a release board when it's made, or is it like, per month you add them?

Reply:
No, there will be a 2 month rolling window for the release boards, so only tasks planned for those releases will be on boards. If you have an issue for GLOWS L2 that you're not ready to work on in the Oct or Nov, then that issue will only be tracked by you under your GLOWS L2 high-level task, not on any boards. The goal is to keep track of issues (under the high-level tasks) without having to plan when you'll work on them too far ahead of time with the boards.

I will try to make that more clear in the documentation.

Reply:
Sounds good, yeah I'd add a bullet point that they are not necessarily part of a release